packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -753,6 +753,23 @@ mixin SchedulerBinding on BindingBase {
   ///  * [scheduleFrameCallback], which registers a callback for the start of
   ///    the next frame.
   void addPostFrameCallback(FrameCallback callback) {
+    assert(() {
+      if (debugTracePostFrameCallbacks) {
+        final String caller = StackTrace.current.toString()
```

Comment:
The StackTrace API format is technically unspecified, and this plus regex is a bit scary to me. Would it be so bad to use a generic event name? or allow developers to speciy a label?

Reply:
I added a debug label